### PR TITLE
[#155824086] Creates a VPN to connect the test environment to pagoPA

### PR DESCRIPTION
**Pull request by cloudify**
Since I had to make a few changes to the configuration that required the recreation of the resources, I've decided to move everything to Terraform.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Remember to fix the PR title by removing WIP wording when ready
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#155824086](https://www.pivotaltracker.com/story/show/155824086): Va creata una VPN per connettere l'ambiente cloud Azure a pagoPA

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: infrastructure/modules/azurerm/kubernetes/main.tf [hunk 1]
```diff
@@ -36,15 +36,25 @@ resource "azurerm_kubernetes_cluster" "azurerm_kubernetes_cluster" {
 }
 
 locals {
+  # This ID gets used by AKS to name resources, there's no current way of
```
Comment (gunzip) [inline infrastructure/modules/azurerm/kubernetes/main.tf]: maybe we can force this value to a know one (so there's no need to get it later) ?

Comment (cloudify) [reply infrastructure/modules/azurerm/kubernetes/main.tf]: haven't seen a way in the docs to do that - I believe something can be done via a `data` module that fetches the virtual networks, anyway it looks like some behavior that will change as the product evolves

File: infrastructure/modules/pagopa-vpn/main.tf [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+  count = "${var.enable == "true" ? 1 : 0}"
+
+  name                  = "${local.loadbalancer_vm_name}"
+  location              = "${var.resource_group_location}"
+  resource_group_name   = "${var.resource_group_name}"
+  network_interface_ids = ["${azurerm_network_interface.lb_interface.id}"]
+
+  # This VM will be mostly idle, we can use a burstable type
+  # https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-general#b-series
+  vm_size = "Standard_B1s"
```
Comment (gunzip) [inline infrastructure/modules/pagopa-vpn/main.tf]: or we can use F1S :D

**Pull request merged**